src/status_im/ui/screens/accounts/recover/views.cljs

```diff
@@ -3,30 +3,54 @@
                     :as views])
   (:require [re-frame.core :as re-frame]
             [reagent.core :as reagent]
+            [status-im.native-module.core :as status]
             [status-im.ui.components.text-input.view :as text-input]
             [status-im.ui.components.react :as react]
             [status-im.ui.components.status-bar.view :as status-bar]
             [status-im.ui.components.toolbar.view :as toolbar]
+            [status-im.ui.components.toolbar.actions :as toolbar.actions]
             [status-im.i18n :as i18n]
             [status-im.ui.screens.accounts.recover.styles :as styles]
+            [status-im.ui.screens.chat.photos :as photos]
             [status-im.ui.components.styles :as components.styles]
             [status-im.utils.config :as config]
             [status-im.utils.core :as utils.core]
             [status-im.react-native.js-dependencies :as js-dependencies]
+            [status-im.react-native.resources :as resources]
             [status-im.ui.components.common.common :as components.common]
             [status-im.utils.security :as security]
             [status-im.utils.platform :as platform]
             [clojure.string :as string]))
 
+(defn cancel-link []
+  [react/text
+   {:on-press     #(do (status/stop-node) (re-frame/dispatch [:navigate-to-clean :intro]))
```

Comment:
why do we want to stop node here?

Reply:
@flexsurfer Because the conditions for enabling the Next button in the first recovery screen include `not node-stopped?` -- this code inherited from existing code,